### ISSUE TITLE
ActionController::API ignored

### ISSUE THREAD

**Issue by max-si-m** [BaseSecrete/rorvswild#15]
Is there is a reason why [ActionController::API](https://github.com/BaseSecrete/rorvswild/blob/master/lib/rorvswild/plugin/action_controller.rb#L6) ignored? 
Events aren't tracked by rorvswild.

~~~ruby
class ApplicationController < ActionController::Base
end

class Api::ApplicationController < ActionController::API
end

ApplicationController.ancestors.include?(ActionController::Base) #=> true
Api::ApplicationController.ancestors.include?(ActionController::Base) #=> false
~~~

As I found callbacks implemented in [AbstractController::Callbacks](https://github.com/rails/rails/blob/master/actionpack/lib/abstract_controller/callbacks.rb#L198)

~~~ruby
Api::ApplicationController.ancestors.include?(AbstractController::Callbacks) #=> true
ApplicationController.ancestors.include?(AbstractController::Callbacks) #=> true
~~~

and after a quick check, I didn't find any restriction.

Thanks a lot, by the way!

**Comment by alexisbernard** [comment]
Thanks for the report. Callbacks were not setup for `ActionController::API`. I published a new version `gem "rorvswild", "1.5.6"` with the fix. Let me know.

**Comment by max-si-m** [comment]
Thanks for so quick response! Looks like everything is working 💪
